azure-pipelines.yml

```diff
@@ -4,10 +4,10 @@
 # Build STL targeting x86, x64, arm, arm64
 
 variables:
-  tmpDir: 'D:\Temp'
-  buildOutputLocation: 'D:\build'
+  tmpDir: 'C:\stl_temp'
+  buildOutputLocation: 'C:\stl_build'

```

Comment:
Any chance these are changing from SSD to spinning rust and slowing our builds?

Reply:
My understanding is that the OS disk is an SSD, but I can't seem to find a source that confirms that.

Reply:
I spose we can examine build times to see if they're in the same ballpark.